lib/hanami/cli/commands/app/db/utils/sqlite.rb

```diff
@@ -17,7 +17,7 @@ def create_command
               end
 
               # @api private
-              def drop_command
+              def drop_command(**)
```

Comment:
seems like "unlink" doesn't support any params, so nothing to do here
https://apidock.com/ruby/Pathname/unlink